<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_2">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="numberLine1">
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string notr="true">Число 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="numberLine2">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Число 2</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>+</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLineEdit" name="resultLine">
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QPushButton" name="pushButton_3">
        <property name="text">
         <string>x</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_4">
        <property name="text">
         <string>:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>753</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>327</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>subtraction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>566</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>600</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>multiplication()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>499</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>division()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>601</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>626</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addition()</slot>
  <slot>subtraction()</slot>
  <slot>multiplication()</slot>
  <slot>division()</slot>
 </slots>
</ui>
